lib/matplotlib/axes/_axes.py

```diff
@@ -6285,6 +6291,10 @@ def pcolorfast(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
                     else:
                         style = "pcolorimage"
             elif x.ndim == 2 and y.ndim == 2:
+                if C.ndim > 2:
+                    raise ValueError('''
```

Comment:
see e.g. https://github.com/matplotlib/matplotlib/blob/17d16f972f43d3cec538bc27f8006ba1913dbc18/lib/matplotlib/axes/_axes.py#L5663-L5666 for how to format a long error message.

Reply:
Fixed in https://github.com/matplotlib/matplotlib/pull/8690/commits/af603a93495230161eb3c2354236f986b15769f8